<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: Poppins;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Holiday Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbHoliday">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>New Year's Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maundy Thursday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Good Friday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Araw ng Kagitingan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Labor Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Independence Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>National Heroes Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bonifacio Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Christmas Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rizal Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese New Year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EDSA Revolution</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black Saturday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ninoy Aquino Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Saints' Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Souls' Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feast of the Immaculate Conception</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Christmas Eve</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Last Day of the Year</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="dateType">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: &quot;#344273&quot;;
	color: white;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
